### ISSUE TITLE
LDMExemptModule.FIRMWARE_UPGRADE not working?

### ISSUE THREAD

**Issue by sjhunn** [dji-sdk/Mobile-SDK-Android-V5#66]
When  I enable the firmware exemption module in LDM:
```
LDMManager.getInstance().enableLDM(context, callback, LDMExemptModule.FIRMWARE_UPGRADE);
```
And try to check for upgrades:
```
UpgradeManager.getInstance().checkUpgradeableComponents(new 
     CommonCallbacks.CompletionCallbackWithParam<ComponentType>() {
            @Override
            public void onSuccess(ComponentType componentType) {
            }
            @Override
            public void onFailure(@NonNull IDJIError error) {
                //fails to here
            }
        });
```
I get the error code: 服务器版本列表还没有拉取到 (The server version list has not been pulled).

If I change the exempt module in the above code to USER_ACCOUNT, I am able to log in.

If I turn off LDM, I am able to call "checkUpgradeableComponents" successfully.

**Comment by DJI-William** [comment]
Confirm. This is a bug.

**Comment by dji-dev** [comment]
This issue has been fixed in version 5.2.

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/76380'>Zendesk ticket #76380</a>:<br><br>Sorry, the fix version is 5.3.
<br><p><em><small>°°°</small></em></p>
